app/src/main/java/com/amaze/filemanager/ui/dialogs/OpenFileDialogFragment.kt

```diff
@@ -385,6 +387,15 @@ class OpenFileDialogFragment : BaseBottomSheetFragment() {
         }
     }
 
+    private fun startActivityCatchingNFC(intent: Intent) {
```

Comment:
I think there are more `Activity`s that would trigger `SecurityException`, like #2329. Better use a more generic name for this.